ldoc/parse.lua

```diff
@@ -72,6 +72,49 @@ local function parse_colon_tags (text)
    return preamble,tag_items
 end
 
+local function parse_lls_tags(text)
+   local lines = stringio.lines(text)
+   local preamble, line = tools.grab_while_not(lines,luadoc_tag)
+   local tag_items = {}
+   local follows
+   while line do
+      local lls_param = '@param%s+([%w_]+)%s([%w_]+)%??%s*#?%s*(.*)'
+      local param_name, param_type, param_desc = line:match(lls_param)
+      local lls_ret = '@return%s+([%w_]+)%s*([%w]*)%s*#?%s*(.*)'
+      local ret_type, ret_name, ret_desc = line:match(lls_ret)
```

Comment:
It's not a bug. If a function returns a variable, you may specify the name of this variable in the `@return` statement of LLS. But I guess there's no use for this variable name in LDoc.

Using `_` instead of `ret_name` is good.